cdci_data_analysis/configurer.py

```diff
@@ -33,89 +33,62 @@
 
 logger = logging.getLogger("conf")
 
-
-# TODO: this looks rather specific to INTEGRAL?
-
 class DataServerConf:
 
-    def __init__(self, data_server_url,
-                 data_server_remote_cache=None,
-                 dispatcher_mnt_point=None,
-                 dummy_cache=None,
```

Comment:
dummy_cache  is not entirely optional. in the sense that we rather want this to be set in production.
it might be useful to allow it to be None in test. But it should at least show a warning, or ideally raise a problem if server is not started in some "debug" mode (the same mode that reports extra details in run_analysis issues)